pinot-spi/src/test/java/org/apache/pinot/spi/config/ConfigUtilsTest.java

```diff
@@ -85,7 +85,7 @@ public void testIndexing() {
         ImmutableMap.of("LOAD_MODE", "MMAP", "AWS_ACCESS_KEY", "default_aws_access_key", "AWS_SECRET_KEY",
             "default_aws_secret_key");
 
-    indexingConfig = ConfigUtils.applyConfigWithEnvVariables(environment, indexingConfig);
+    indexingConfig = ConfigUtils.applyConfigWithEnvVariablesAndSystemProperties(environment, indexingConfig);
```

Comment:
Shall we add some value to the system properties and test out the behavior?

Reply:
added.